src/tests/multihost/alltests/test_rfc2307bis.py

```diff
@@ -0,0 +1,62 @@
+""" Automation of rfc2307bis
+
+:requirement: rfc2307bis
+:casecomponent: sssd
+:subsystemteam: sst_idm_sssd
+:upstream: yes
+:status: approved
+"""
+import os
+import re
+import posixpath
+import pytest
+from sssd.testlib.common.utils import sssdTools
+from constants import ds_instance_name
+
+@pytest.mark.usefixtures('setup_sssd')
+@pytest.mark.rfc2307bis
```

Comment:
rfc2307bis marker can not be found anywhere 

https://github.com/SSSD/sssd/blob/master/src/tests/multihost/alltests/pytest.ini
https://github.com/SSSD/sssd/blob/master/src/tests/multihost/alltests/readme.rst

Reply:
Added

Reply:
added